Cargo.toml

```diff
@@ -44,7 +44,7 @@ lazy_static = { version = "1.4.0", optional = true }
 log = { version = "0.4", optional = true } # Used only in std
 rand = { version = "0.8.5", default-features = false, features = ["alloc", "getrandom"] }
 serde = { version = "1.0", default-features = false, features = ["alloc", "derive"] }
-spin = { version = "0.9", default-features = false, features = ["mutex", "spin_mutex"] } # No_std alternative for std::sync::Mutex
+spin = { version = "0.9.8, default-features = false, features = ["mutex", "spin_mutex"] } # No_std alternative for std::sync::Mutex
```

Comment:
Hi @Dylan-DPC ,

Thanks for reporting and fixing this!
However, as you can see in failing checks, there is a typo in you patch: you deleted the ending `"` of version number.
Once you fix this, I'll be happy to merge your PR 🙂

Reply:
ah oops xD